tests/web/squadre_flow.py

```
@@ -0,0 +1,221 @@
+    pg.click('a.src-btn:has-text("test 1")')
+    pg.wait_for_selector('[data-act="alias-save"]')
+    assert alias_visibili(pg) == {'AC Milan': '', 'Juventus': ''}, \
+        'la tabella parte dalle squadre Betfair della competizione, alias vuoti'
+    pg.fill('[data-nome="Juventus"]', 'Juve')
+    pg.fill('[data-nome="AC Milan"]', 'Milan')
+    pg.click('[data-act="alias-save"]')
+    pg.wait_for_selector('.toast:has-text("Alias salvati")')
+    shot(pg, 'squadre-alias')
+
+    # Il badge si aggiorna: 2/2.
+    pg.click(f'a:has-text("Serie A")')
```

Ruff (0.16.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)